### PR TITLE
GameBuddy macropad

### PR DESCRIPTION
**Pull request by AlisGraveNil**
A 26-key macropad with a focus on gaming

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clawsome/gamebuddy/config.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/config.h:24]:
```suggestion
#define PRODUCT_ID      0x17B9
```
```
echo -n "GameBuddy" | shasum | cut -c1-4
```

File: keyboards/clawsome/gamebuddy/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+This is a 5x7 macropad designed for left-hand usage for videogames, with special focus on FPS layouts!
+
+    Keyboard Maintainer: AAClawson
+    Hardware Supported: GameBuddy, Pro Micro, Elite-C
+    Hardware Availability: www.clawboards.xyz
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/readme.md:5]:
```suggestion
* Keyboard Maintainer: AAClawson
* Hardware Supported: GameBuddy, Pro Micro, Elite-C
* Hardware Availability: www.clawboards.xyz
```

File: keyboards/clawsome/gamebuddy/readme.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+This is a 5x7 macropad designed for left-hand usage for videogames, with special focus on FPS layouts!
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/readme.md:1]:
```suggestion
# GameBuddy

This is a 5x7 macropad designed for left-hand usage for videogames, with special focus on FPS layouts!
```

File: keyboards/clawsome/gamebuddy/readme.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+    Keyboard Maintainer: AAClawson
+    Hardware Supported: GameBuddy, Pro Micro, Elite-C
+    Hardware Availability: www.clawboards.xyz
+
+Make example for this keyboard (after setting up your build environment):
+
+make clawsome/gamebuddy:default
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/readme.md:9]:
```suggestion
    make clawsome/gamebuddy:default
```

File: keyboards/clawsome/gamebuddy/readme.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+See the build environment setup and the make instructions for more information. Brand new to QMK? Start with our Complete Newbs Guide.
+
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/readme.md:12]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/clawsome/gamebuddy/rules.mk [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```
Comment (drashna) [inline keyboards/clawsome/gamebuddy/rules.mk:5]: If this is pro micro compatible/powered, then ideally this should be caterina by default.

Comment (AlisGraveNil) [reply keyboards/clawsome/gamebuddy/rules.mk:5]: @drashna so what do I need to change?

File: keyboards/clawsome/gamebuddy/readme.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# GameBuddy
+
+This is a 5x7 macropad designed for left-hand usage for videogames, with special focus on FPS layouts!
+
+* Keyboard Maintainer: AAClawson
```
Comment (noroadsleft) [inline keyboards/clawsome/gamebuddy/readme.md:5]:
```suggestion
* Keyboard Maintainer: [AAClawson](https://github.com/AlisGraveNil)
```

File: keyboards/clawsome/gamebuddy/rules.mk (hunk 6 above)
Comment (noroadsleft) [reply keyboards/clawsome/gamebuddy/rules.mk:5]:
```suggestion
BOOTLOADER = caterina
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**